docs/content/references/sui-sdks.mdx

```diff
@@ -1,42 +1,49 @@
 ---
-title: Sui SDKs
+title: Sui and Community SDKs
+description: Collection of SDKs and utilities for developing on Sui using various programming languages.
 ---
 
-Sui provides developer kits that act as wrappers for the Sui API.
-
-## dApp Kit
-
-[Sui dApp Kit](https://sdk.mystenlabs.com/dapp-kit) is a web frontend SDK that interacts with the Sui API. It is available as an NPM package.
-
-## Dart SDK
-
-[Sui Dark SDK](https://github.com/mofalabs/sui) is a cross-platform Sui SDK for Mobile, Web, and Desktop.
-
-## Go SDK
-
-For those that like gophers, see [Go SDK](https://github.com/block-vision/sui-go-sdk).
-
-## Kotlin SDK
-
-[Ksui](https://github.com/mcxross/ksui) is a collection of Kotlin Multiplatform JSON-RPC wrapper and crypto utilities for interacting with a Sui Full node.
-
-## Python SDK
-
-For snake lovers, see [Python SDK](https://github.com/FrankC01/pysui).
-
-## Rust SDK
-
-See [Rust SDK](./rust-sdk.mdx) for SDK configuration and examples of using the Sui API with Rust.
-
-## Swift SDK
-
-[SuiKit](https://github.com/opendive/suikit) is a Swift SDK natively designed to make developing for the Sui Blockchain easy.
-
-## TypeScript SDK
-
-The Sui TypeScript SDK documentation is available in its own microsite: [Sui Typescript SDK](https://sdk.mystenlabs.com/typescript).
-
-
-### Crate documentation
-
-[GitHub Pages](https://mystenlabs.github.io/sui/sui_sdk/index.html) hosts the generated documentation for Rust crates in the Sui repository.
+Sui provides developer kits that act as wrappers for the Sui API. The Sui community broadens the code coverage with its own set of developer kits targeting the Sui blockchain. 
+
+## Sui SDKs
+
+<Cards>
+<Card title="dApp Kit" href="https://sdk.mystenlabs.com/dapp-kit">
+    A web frontend SDK that interacts with the Sui API. It is available as an NPM package.
+</Card>
+<Card title="Rust SDK" href="./rust-sdk">
+    SDK configuration and examples of using the Sui API with Rust.
+</Card>
+<Card title="Sui-sdk crate" href="https://mystenlabs.github.io/sui/sui_sdk/index.html">
+    Auto-generated documentation for the Rust `sui-sdk` crate in the Sui repository.
+</Card>
```

Comment:
Thoughts on folding this as a link in the "Rust SDK" card as it's the auto-generated docs for that SDK?

Reply:
I almost didnt do the cards because of this. It technically works, but a missed click on the internal link puts the user in a different location (as it's essentially a link inside a link). Might be a low risk but I'd like to update the element to account for this in a more user friendly way. Could either leave as is or fold until it's updated. I'd rather leave as is for now just because pushing to github is so difficult :)

Reply:
Here's how that looks: https://sui-core-b2hs54tsc-sui-foundation.vercel.app/references/sui-sdks. I don't hate it. At least until I can update.

Reply:
Well, it does open both. The sui-sdk opens in a new tab and the Rust page opens in the same tab.

Reply:
Yeah that's a bit janky -- having a separate card works -- let me see if I can tweak the wording to make it clearer that the two things are related?